### DISCUSSION TITLE
CI Workflow Not Detecting Tests in Python: Has Anyone Found a Solution?

### DISCUSSION THREAD

**Discussion by Awaab98** [MIT-Emerging-Talent/ET6-foundations-group-12#15] [Q&A]
Hi everyone,

I’ve been facing a recurring issue with the CI check tests in our repository. Whenever I push changes to my branch, the tests don’t seem to run, and the program fails to detect them. This has made it extremely difficult to pass the CI checks.

I’ve attached some screenshots showing the issue, including how the tests aren’t being detected in the workflow. Has anyone else encountered this problem? If so, were you able to resolve it?

I suspect there might be a misconfiguration in the CI workflow or test files, but I’m unsure how to pinpoint or fix it. Any insights or suggestions would be greatly appreciated!

![Screenshot 2024-12-27 170923](https://github.com/user-attachments/assets/f81b393b-b0c9-47ef-945c-7e70d20e2feb)
![image](https://github.com/user-attachments/assets/5f6ded4e-16ae-480d-8739-91593dd2b2d1)

**Comment by Awaab98** [comment]
@abeddost @Yevheniia-Rudenko

**Comment by Yevheniia-Rudenko** [comment]
@Awaab98 Sorry for the late respond! I belive the issue is solved, right?
